bin/cmds/create.js

```diff
@@ -9,67 +9,62 @@ const fs = require('fs')
 const fabricator = require('../../lib')
 const bistre = require('bistre')()
 
-exports.command = 'create [options]'
+exports.command = 'create'
 exports.desc = 'performs full provisioning of a Kubernetes cluster and deployment of software.\n' +
 'It will fetch defaults from a configured Vault server, and store results in Vault.\n' +
 'Defaults can be overridden as extra yargs arguments, e.g.:\n\n' +
 '--arg-organizationId 1234567890 or --arg-cluster.worker.memory 26GB'
-exports.build = {
-  url: {
-    description: 'the url of the cluster you wish to create, e.g. `mycluster.example.com`',
-    alias: 'u'
-  },
-  name: {
-    description: 'the name, or general identifier of the cluster.  Can be inferred from the url',
-    alias: ['n']
-  },
-  slug: {
-    description: 'the general identifier of the cluster.  Can be inferred from the url.  ' +
+exports.builder = function (yargs) {
+  return yargs
+    .option('url', {
```

Comment:
Any reason why you switched to individual `option`s rather than a single `options` object?

Reply:
Nope, this was lazy search/replacing in full force here.